content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Istio CA Integration with SPIRE 
+linktitle: How to integrate Istio with SPIRE through Envoy SDS API.
+description: Describes how to configure Istio to integrate with SPIRE to get cryptographic identities through Envoy's SDS API.
+weight: 40
+keywords: [kubernetes,spiffe,spire]
+aliases:
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE specification that performs node and workload attestation in order to securely issue cryptographic identities to workloads running in heterogeneous environments.  

```

Comment:
The hard paragraph breaks need to be fixed in a couple places (see attached screenshot). I think the problem is that Markdown interprets two or more spaces at EOL plus a line break as an HTML `<br>` character. So if you just delete the two spaces from the end of the two affected lines, it should fix the problem. Besides this line, the next one with spaces at EOL starts with "SPIRE Agent can be configured to push..."

<img width="782" alt="Screen Shot 2022-03-15 at 11 54 18 AM" src="https://user-images.githubusercontent.com/54969501/158452547-76df5f6c-1043-4d1f-b5f4-decbe705c8e4.png">

Reply:
Got it!